Creategoalpanel1

franarave opened the pull request:
CreGo1M - Create Goal - Panel 1 [Mobile] #1458 Review

franarave committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into RangeSelectionCalendar

franarave committed:
CreGo1M - Create Goal - Panel 1 [Mobile] #1458 Review

https://github.com/DuckyTeam/ducky-web/issues/1458

nathan-bird commented on src/CreateGoalPanel1/index.js:
```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import classNames from 'classnames';
```
classNames isn't used at the moment. You can add it to the first div className

franarave replied:
Done

nathan-bird commented on src/CreateGoalPanel1/index.js:
```diff
@@ -0,0 +1,84 @@
+
+        <div className={styles.outerWrapper}>
+              <SectionHeaderGeneral
+                      onClick={props.onCancel}
```
onCancel isn't part of your props validation (the propTypes object at the end of your component). The component works fine without it however having a propType ensures the component is used as intended. You can read more [here](https://facebook.github.io/react/docs/reusable-components.html)

nathan-bird commented on src/CreateGoalPanel1/stories.js:
```diff
@@ -0,0 +1,17 @@
+import {bool, stories, oneOf, func, string, number, className} from '../duckyStories';
```
oneOf and number are not used

nathan-bird commented on src/CreateGoalPanel1/stories.js:
```diff
@@ -0,0 +1,17 @@
+  children: string('Bygg en vane eller fullfør prestasjoner iløpet av en fastsatt periode.'),
+  title: string('Sett nytt personlig mål'),
+  onClick: func(),
+  iconClicked: bool(),
```
iconClicked not used in the component

nathan-bird commented on src/CreateGoalPanel1/index.js:
```diff
@@ -0,0 +1,84 @@
+    inactive: React.PropTypes.bool,
+    onClick: React.PropTypes.func,
+    type: React.PropTypes.string,
+    size:React.PropTypes.string
```
size isnt used as a prop

nathan-bird commented on src/CreateGoalPanel1/index.js:
```diff
@@ -0,0 +1,84 @@
+              <SectionHeaderGeneral
+                      onClick={props.onCancel}
+                      rightIcon={"icon-close"}
+                      title={props.title}
```
title is also missing in props validation

nathan-bird reviewed (changes requested):
Great component.

I would stress to try and conform to es-lint styling guidelines to improve the readability of your code.
At the moment this component shows 85 es-lint errors. These don't break your component (although some do), but resolving them makes the code much easier to understand and helps anyone else looking at it.
Install the atom packages "linter" and "linter-eslint" to highlight the errors. Follow this guide to get the packages installed (http://flight-manual.atom.io/using-atom/sections/atom-packages/). If you need any help just ask :)

franarave committed:
SwiLab01

Toggle button with label #1586
Fixing SwitchToogleButton issue
Fixing NotificationItem
Fixing SectionFooterNew

franarave committed:
CreateGoalPanel1

fixing component's problems
change in ActionSelectComposit styles.css
change in CreateGoalActionItem styles.css

nathan-bird committed:
spelling
